src/dbt_score/formatters/human_readable_formatter.py

```diff
@@ -44,26 +44,34 @@ def evaluable_evaluated(
         """Callback when an evaluable item has been evaluated."""
         if score.value < self._config.fail_any_item_under:
             self._failed_evaluables.append((evaluable, score))
+        if score.value < self._config.fail_any_item_under or self._config.show in [
+            "all",
+            "failing-rules",
+        ]:
```

Comment:
Currently, when linting my dbt project, the headers of models that pass are still shown when using `failing-rules`. e.g. 🥇 M: model_name (score: 10.0), without the rules.

So we have to change the if statements a bit!

Reply:
you are right. I changed the if condition and tested it again with `tests/resources/manifest.json` . Now it does not print the evaluable's header if not failing and no failing rules.